### ISSUE TITLE
Release the code?

### ISSUE THREAD

**Issue by wingvortex** [mylonasc/wind-farm-graph-dataset#1]
Hi, the paper seems very interesting, could you share the source code?

**Comment by mylonasc** [comment]
Of course! Glad you asked! If you're in a hurry check the supporting materials from [here](https://openreview.net/forum?id=mrsx7ninrtU) (the zip file). I'll upload a baseline notebook soon.

**Comment by wingvortex** [comment]
Thanks for sharing the link to the code and supplemental material!
